certified-connectors/Nosco/apiProperties.json

```diff
@@ -0,0 +1,85 @@
+{
+  "properties" : {
+    "connectionParameters" : {
+      "api_key" : {
+        "type" : "securestring",
+        "uiDefinition" : {
+          "displayName" : "API Key",
+          "description" : "The API key used to connect to the Nosco platform",
+          "tooltip" : "Provide your API key",
+          "constraints" : {
+            "tabIndex" : 2,
+            "clearText" : false,
+            "required" : true
+          }
+        }
+      },
+      "platform_host" : {
+        "type" : "string",
+        "uiDefinition" : {
+          "displayName" : "Platform hostname",
+          "description" : "The hostname of the Nosco platform",
+          "tooltip" : "Provide the hostname of your Nosco platform",
+          "constraints" : {
+            "required" : true
+          }
+        }
+      }
+    },
+    "iconBrandColor" : "#131313",
+    "publisher" : "Nosco ApS",
+    "stackOwner" : "Nosco ApS",
```

Comment:
stackOwner should be Name of the owner of the service or product.

Reply:
Nosco is both the publisher and service owner, and we are trying to publish a verified connector. According to the following documentation, that is the correct value for the stackOwner field:

https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3a-publisher-and-stackowner-properties